coderd/deprecated.go

```diff
@@ -70,3 +75,42 @@ func (api *API) workspaceAgentLogsDeprecated(rw http.ResponseWriter, r *http.Req
 func (api *API) workspaceAgentsGitAuth(rw http.ResponseWriter, r *http.Request) {
 	api.workspaceAgentsExternalAuth(rw, r)
 }
+
+// @Summary Removed: Submit workspace agent metadata
+// @ID removed-submit-workspace-agent-metadata
+// @Security CoderSessionToken
+// @Accept json
+// @Tags Agents
+// @Param request body agentsdk.PostMetadataRequestDeprecated true "Workspace agent metadata request"
+// @Param key path string true "metadata key" format(string)
+// @Success 204 "Success"
+// @Router /workspaceagents/me/metadata/{key} [post]
+// @x-apidocgen {"skip": true}
+func (api *API) workspaceAgentPostMetadataDeprecated(rw http.ResponseWriter, r *http.Request) {
```

Comment:
Would it be worthwhile dropping a WARN log advising operators to restart the workspace?

Reply:
Good question. What if it's a workspace that is running something semi-critical and can't be shut down for a week? We'd potentially be spamming WARN every second :/

Reply:
Fair point!